ctapipe/io/tests/test_hdf5.py

```diff
@@ -59,58 +84,52 @@ class C2(Container):
         d = Field('d', None)
 
     with tempfile.NamedTemporaryFile() as f:
```

Comment:
You should not use `tempfile` in tests, and instead of pytest's built-in fixtures for that.  
I think pytest recommends using it's own system, since tempfile may not do the right thing in self-contained testing sessions, if I remember right. The correct usage is here:  
https://docs.pytest.org/en/latest/tmpdir.html

Reply:
done